app.rb

```diff
@@ -14,6 +14,9 @@ class App < Sinatra::Base
 
   configure do
     set :log_level, Logger::DEBUG
+
+    set :gateways,
+      send_sms: WifiUser::Gateway::GovNotifySMS.new(ENV.fetch('NOTIFY_API_KEY'))
```

Comment:
for my peace of mind (+ others unsure), the use cases can depend on the gateways by doing this:

```ruby
set :use_cases, -> do
  {
    some_usecase: SomeUsecase.new(gw: settings.gateways.some_gateway)
  }
end
```